utils/nctl/sh/scenarios-upgrades/upgrade_scenario_08.sh

```diff
@@ -98,17 +99,33 @@ function _step_05()
     log "... allowing 15 seconds for the node to report errors"
     sleep 15
 
-    COUNT=$(cat "$NCTL"/assets/net-1/nodes/node-"$NODE_ID"/logs/stdout.log 2>/dev/null | grep 'peer is running incompatible version' | wc -l)
+    local COUNT=$(cat "$NCTL"/assets/net-1/nodes/node-"$NODE_ID"/logs/stdout.log 2>/dev/null | grep 'peer is running incompatible version' | wc -l)
     if [ "$COUNT" -eq "0" ]; then
         log "ERROR: We got 0 error messages, but expected some"
         exit 1
     fi
 }
 
-# Step 06: Terminate.
+# Step 06: Assert joiner node is not connected to any peers
 function _step_06()
 {
-    log_step_upgrades 6 "test successful - tidying up"
+    local NODE_ID=${1}
+
+    log_step_upgrades 6 "asserting node has no peers connected"
+
+    local COUNT=$(curl -s "$(get_node_address_rest "$NODE_ID")/status" | jq '.peers' | jq length)
```

Comment:
If it's not too difficult to move that up into NCTL tooling then it would be great - something like `get_node_peers` ?

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/2577/commits/8b18fe0f1d33510790b27b37c86cae1165ebc17f

Reply:
Thanks!